# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../simgear/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur SimGear-0.3.9-orig/simgear/scene/model/placement.hxx SimGear-0.3.9/simgear/scene/model/placement.hxx
--- SimGear-0.3.9-orig/simgear/scene/model/placement.hxx	2005-09-13 00:45:58.000000000 +0000
+++ SimGear-0.3.9/simgear/scene/model/placement.hxx	2006-04-04 15:36:11.868207500 +0000
@@ -46,7 +46,7 @@
   SGModelPlacement ();
   virtual ~SGModelPlacement ();
 
-  virtual void SGModelPlacement::init( ssgBranch * model );
+  virtual void init( ssgBranch * model );
 
   virtual void update();
 
